docs/book/src/migration/v1vsv2.md

```diff
@@ -0,0 +1,61 @@
+# Kubebuilder v1 vs v2
+
+There are a number of changes in
+[controller-runtime](https://github.com/kubernetes-sigs/controller-runtime),
+[controller-tools](https://github.com/kubernetes-sigs/controller-tools)
+and [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder).
+
+## Common changes
+
+- They have migrated from `dep` to `go modules` for managing dependencies.
+
+## Controller-runtime
+
+- Automatic certificate generation for webhooks has been removed, and webhooks
+will no longer self-register. Use controller-tools to generate a webhook
+configuration, and use something like cert-manager to generate your certificates
+(if you don't already have a solution). Kubebuilder supports them out-of-the-box.
```

Comment:
Lang: In `KubeBuilder supports them`, it's unclear what `them` is.  certmanager is a thing, so it should be `it`, if that's what you're talking about